Bottom: 1d854efb4a66bf8cc596784039776771644f0039
Top:    b7ac0132a89eb8dccd5ec1095b950db9b17b3243
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 14:05:27 +0530

Refresh of sched-fair-provide-arch-hook

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index a32a3d9b5ec5..27387a4e0543 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5926,8 +5926,8 @@ static int select_non_idle_core(struct task_struct *p, int prev_cpu, int target)
 	struct cpumask *cpus = this_cpu_cpumask_var_ptr(turbo_sched_mask);
 	int iter_cpu, sibling;
 
-	cpumask_and(cpus, cpu_online_mask, arch_turbo_domain(prev_cpu));
-	cpumask_and(cpus, cpus, p->cpus_ptr);
+	cpumask_and(cpus, cpu_online_mask, p->cpus_ptr);
+	cpumask_and(cpus, cpus, arch_turbo_domain(prev_cpu));
 
 	for_each_cpu_wrap(iter_cpu, cpus, prev_cpu) {
 		int idle_cpu_count = 0, non_idle_cpu_count = 0;
